source/agora/test/Base.d

```diff
@@ -553,6 +553,23 @@ public class TestAPIManager
                     idx, node.getBlockHeight(), height), file, line));
     }
 
+    public void expectBlock (Height height, const(BlockHeader) enrolled_header,
```

Comment:
This needs documentation. Actually if it only checks the enrollments then it might be better to just pass `b0.header.enrollments` instead as the parameter.

Reply:
In fact, it needs `height` because the `waitForPreimages` in the function is called with a distance which is the requested height minus the enrolled height. I documented that.